modules/33acrossIdSystem.js

```diff
@@ -9,32 +9,42 @@ import { logMessage, logError } from '../src/utils.js';
 import { ajaxBuilder } from '../src/ajax.js';
 import { submodule } from '../src/hook.js';
 import { uspDataHandler, coppaDataHandler, gppDataHandler } from '../src/adapterManager.js';
+import { getStorageManager, STORAGE_TYPE_COOKIES, STORAGE_TYPE_LOCALSTORAGE } from '../src/storageManager.js';
+import { MODULE_TYPE_UID } from '../src/activities/modules.js';
 
 const MODULE_NAME = '33acrossId';
 const API_URL = 'https://lexicon.33across.com/v1/envelope';
 const AJAX_TIMEOUT = 10000;
 const CALLER_NAME = 'pbjs';
+const GVLID = 58;
```

Comment:
if you're going to abort in GDPR countries, why list a gvlid?

Reply:
see #10495

Reply:
If we remove the GVLID, it looks like it will affect the scenario where all of the following true:
1. gdprApplies=true
2. gdprEnforcement module is part of the Prebid.js build
3. 33across GVLID is part of the list of allowed vendors
4. Country is not in the EU
In theory the publisher's CMP shouldn't indicate that GDPR applies outside of the EU, but in practice I'm not sure whether CMPs reliably report this. Personally I don't see any harm in keeping the GVLID, but if you feel strongly that we should remove it then we'll go ahead and do so.

Reply:
you need some way of preventing network traffic in regimes you don't support. Feel free to keep the gvlid and detect gdpr applies and suppress the call.

Reply:
@patmmccann 

We have made the corresponding changes and now the ID module suppresses the calls where GDPR applies.

Changes have been incorporated to this PR and once approved they should fix [#10495](https://github.com/prebid/Prebid.js/issues/10495)